src/components/Card.jsx

```diff
@@ -1,53 +1,85 @@
-<button  className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-red-400 border border-[#0A2E50] p-3">BLACK</button>
-<button  className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-green-400 border border-[#0A2E50] p-3">ASIAN</button>
-<button  className="bg-yewhite-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-yellow-300 border border-[#0A2E50] p-3">DISABILITY</button>
-<button  className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-purple-400 border border-[#0A2E50] p-3">LATIN</button>
-<button  className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-pink-400 border border-[#0A2E50] p-3">IMMIGRANT</button>
-
-</div>
-    <div className="w-full py-[10rem] px-4 bg-white ">
-      <div className="max-w-[1040px] grid md:grid-cols-3 gap-12 place-items-center m-auto  ">
-        {details}
+          </button>
+        </div>
+      </form>
+      <div className="flex items-center justify-center mt-4 m-auto">
+        <button className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5 rounded-[20px] dark:bg-gray-700 dark:text-blue-400 border border-[#0A2E50] p-3">
+          LGBTQ+
+        </button>
+        <button className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-gray-400 border border-[#0A2E50] p-3">
+          Women
+        </button>
+        <button className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-red-400 border border-[#0A2E50] p-3">
+          Black
+        </button>
+        <button className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-green-400 border border-[#0A2E50] p-3">
+          Asian
+        </button>
+        <button className="bg-yewhite-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-yellow-300 border border-[#0A2E50] p-3">
+          Disability
+        </button>
+        <button className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-purple-400 border border-[#0A2E50] p-3">
+          Latin
+        </button>
+        <button className="bg-white-100 text-[#0A2E50] text-xs font-medium mr-2 px-2.5 py-0.5  rounded-[20px] dark:bg-gray-700 dark:text-pink-400 border border-[#0A2E50] p-3">
+          Immigrant
+        </button>
+      </div>
+      <div className="w-full py-[10rem] px-4 bg-white ">
+        <div className="max-w-[1040px] grid md:grid-cols-3 gap-12 place-items-center m-auto  ">
```

Comment:
something is still a bit off on the responsiveness - when i shrink the screen right before it's small enough to render as a single column there's overlap 
![Screenshot 2023-11-02 at 11 51 23 AM](https://github.com/cherryontech/unmatched/assets/115492619/718b1e4f-a544-4bff-aab8-e4f59fed692b)

Also at the biggest screen size on my external monitor I'm not seeing the same spacing that shows up in your attached screenshot
![Screenshot 2023-11-02 at 11 52 42 AM](https://github.com/cherryontech/unmatched/assets/115492619/b8d129c5-34df-4e38-9646-33e1f61c08c0)

Reply:
responsiveness will be adressed hopefully in the next PR! :)